websites/documentation/src/content/writing/quizzes.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Learning Plugin
+wideLayout: true
+navLevels: 2
+---
+
+# commercetools learning plugin
+
+A gatsby theme plugins providing UI components and helper utilities to integrate self-learning functionality
+into other React User interfaces.
```

Comment:
Well, it's a gatsby plugin so it can't really be used to put self learning into any React interfaces.  It's specifically for putting them into docs kit websites and will rather get much more specific over time.

Reply:
true .. I guess I just moved the Readme without reading the full content again.. I'll adjust it